test/integration/targets/lookup-option-names/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
The integration test needs some work. It doesn't verify the issue and can be simplified:

```yaml
- debug:
    msg: "{{ lookup('vars', name='test') }}"

- debug:
    msg: "{{ query('vars', name='test') }}"
```